### ISSUE TITLE
Feature to have tabs full width of header

### ISSUE THREAD

**Issue by hkhaka** [maykar/custom-header#145]
When using Custom Header, the scroll "field" for the view gets a lot more narrow than usual (without Custom Header)
Latest and greatest versions of everything (HA, Custom Header and Android app)

First picture is standard HA app layout.
![Normal](https://user-images.githubusercontent.com/45568793/76263292-cfbded00-625e-11ea-9dda-55d41649c473.jpg)

Second picture is with Custom Header enabled:
![Custom header](https://user-images.githubusercontent.com/45568793/76263343-ee23e880-625e-11ea-9210-01f69e5ad7f9.jpg)

**Comment by hkhaka** [comment]
HA version: 0.106.5
Android App version: 1.8.0-144
Custom Header: 1.3.2

'''custom_header:
  exceptions:
    - conditions:
        user: 232d473cf7c942019bd10f482e93e7e4
      config:
        disable_sidebar: true
        hide_config: true
        hide_help: true
        hide_raw: true
        hide_unused: true
        menu_hide: true
        options_hide: true
        show_tabs: ellas
        voice_hide: true
    - conditions:
        user: 145eac3770944160af5a689aa2f37c3e
      config:
        disable_sidebar: true
        hide_config: true
        hide_help: true
        hide_raw: true
        hide_unused: true
        menu_hide: true
        options_hide: true
        show_tabs: siris
        voice_hide: true
  shadow: false
resources:
  - type: module
    url: /community_plugin/custom-header/custom-header.js

**Comment by maykar** [comment]
2nd part of this post:

https://community.home-assistant.io/t/custom-header/155399/473

TLDR: This is a feature, but may at some point be configurable

**Comment by hkhaka** [comment]
OK, I get it.
Thanks for the info. 
Can live with it for now.
I explicitly disabled "Display Tab Chevrons" at least, to get a bit more space.

But I have heard about people stopped using Custom Headers, just because of this.

Great plugin though! Solves a bunch of other things for me. 
Keep up the good work!

**Comment by maykar** [comment]
No need to close this one, going to change it to a feature request.

**Comment by maykar** [comment]
Added in latest version by default

https://community.home-assistant.io/t/custom-header/155399/636
